ports/servoshell/desktop/tracing.rs

```diff
@@ -115,58 +101,3 @@ mod from_winit {
         }
     }
 }
-
-mod from_servo {
-    use super::LogTarget;
-
-    macro_rules! target {
-        ($($name:literal)+) => {
-            concat!("servoshell<servo@", $($name),+)
-        };
-    }
-
-    impl LogTarget for servo::EmbedderMsg {
```

Comment:
We should also update the [documentation](https://book.servo.org/hacking/debugging.html?highlight=trace#event-tracing) on event tracing, I think.  Could libservo still have some support for logging the events that it fires on the delegates?